src/components/JobDetails.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <h1>JobDetails</h1>
+    <h2>Job id : {{id}}</h2>
+    <h2>DETAILS</h2>
+    <h3>{{details}}</h3>
+    </template>
+    
+    <script>
+    import {
+        jobs
+    } from './jobs';
+    export default {
+        name: 'JobDetails',
+        props: {
+            id: String
```

Comment:
@Priya-Sandal Can you user routing param here, instead of props
https://router.vuejs.org/guide/essentials/passing-props.html